src/components/Card.tsx

```diff
@@ -0,0 +1,22 @@
+import { Component } from "react";
+import { Character } from "../types/types";
+
+export class Card extends Component<Character> {
+  render() {
+    const { name, image, gender, species, status } = this.props;
+
+    return (
+      <li className="card">
+        <p className="card-name">
+          <b>{name}</b>
+        </p>
+        <div>
+          <img className="card-image" src={image} alt={name} />
```

Comment:
я не понял зачем див
```suggestion
        <div>
          <img className="card-image" src={image} alt={`Portrait of ${name}`} />
```